Makefile

```diff
@@ -0,0 +1,18 @@
+TARGET_DIR?=$(shell pwd)
+GITHUB_GROUP=github.com/th2-net
+
+TH2_GRPC_COMMON=th2-grpc-common
+TH2_GRPC_COMMON_URL=$(GITHUB_GROUP)/$(TH2_GRPC_COMMON)@makefile
```

Comment:
I think that maybe we should use submodules here. Because downloading sources each time won't make it possible to use this as a library. Also, with submodules we will see what is the current state of proto files